refapp/SKWebApi/appsettings.json

```diff
@@ -1,22 +1,35 @@
 {
-/*
-The following identity settings need to be configured
-before the project can be successfully executed.
-For more info see https://aka.ms/dotnet-template-ms-identity-platform
-*/
+  "CompletionConfig": {
+    "Label": "Completion",
+    "AIService": "AzureOpenAI",
+    "DeploymentOrModelId": "text-davinci-003",
+    "Endpoint": "https://YOUR-INSTANCE.openai.azure.com/",
+    "Key": "DONT_CHECK_IN_THIS_KEY"
+  },
+  // Remove or comment out EmbeddingConfig if no embedding backend is required
+  "EmbeddingConfig": {
+    "Label": "Embeddings",
+    "AIService": "AzureOpenAI",
+    "DeploymentOrModelId": "text-embedding-ada-002",
+    "Endpoint": "https://YOUR-INSTANCE.openai.azure.com/",
+    "Key": "DONT_CHECK_IN_THIS_KEY"
+  },
+  /* The following identity settings need to be configured
+     before the project can be successfully executed.
+     For more info see https://aka.ms/dotnet-template-ms-identity-platform */
   "AzureAd": {
```

Comment:
[nit] - I recommend keeping values to be set by the developer as blank and have a comment next to them showing the format. This makes sure that the system will fail with empty config values unless they are set, rather than trying to go to invalid URLs using invalid GUIDs.

Reply:
Done